test/events.test.ts

```diff
@@ -185,6 +185,42 @@ test("should call handler after tilesloaded when addListener", () => {
   expect(handlerSpy).toHaveBeenCalledTimes(1);
 });
 
+test("should call handler after center_changed when addListenerOnce", () => {
+  // mock map so that we can mock on so that we can mock center_changed
```

Comment:
```suggestion
  // mock map so that we can mock center_changed
```

Reply:
Thanks! I will fix these as well the instances I found in another test file as well